src/serialization/preprocessor.cpp

```diff
@@ -48,6 +48,8 @@ static bool encode_filename = true;
 
 static std::string preprocessor_error_detail_prefix = "\n    ";
 
+static const char OUTPUT_SEPERATOR = '\376';
```

Comment:
It's sep**a**rator, not sep**e**rator.

Also, I'd prefer using a hex escape code: `'\xFE'`